.github/workflows/lint.yml

```diff
@@ -29,17 +29,21 @@ jobs:
           key: yarn-${{ hashFiles('**/yarn.lock') }}
           restore-keys: |
             yarn-
-
+            
       - name: Install yarn dependencies
-        run: yarn install --no-progress --non-interactive
+        run: yarn install --no-progress --non-interactive 
 
-      - name: Set up python
+      - name: Set up Python 3.7
```

Comment:
Do we need to set up python in order to run the linter ? I don't see any Python code in our repo atm, but am I missing something ?

Reply:
ok I see this now : https://github.com/snapshot-labs/sx-core/blob/vanilla_vote/test/starknet/test_contract.py

Reply:
cairo-format requires the python env i think